model-optimizer/unit_tests/extensions/back/MarkNodesWithShapeValues_test.py

```diff
@@ -9,9 +9,8 @@
 from mo.front.common.partial_infer.utils import int64_array, float32_array
 from mo.graph.graph import Node
 from mo.utils.ir_engine.compare_graphs import compare_graphs
-from mo.utils.unittest.graph import build_graph
-from mo.utils.unittest.graph import result, regular_op_with_empty_data, \
-    shaped_const_with_data, connect, regular_op
+from unit_tests.utils.graph import build_graph, result, regular_op_with_empty_data, shaped_const_with_data, connect, \
```

Comment:
In unit_tests folder we will not create mo subfolder? Or just utils will be not in mo?

Reply:
I actually don't see a reason to create additional directory "mo" for utils-like modules. Because the unit tests already belong to MO only so one more depth in the tree doesn't seem reasonable for me.